app/forms/declaration/DeclarantDetails.scala

```diff
@@ -25,7 +28,18 @@ object DeclarantDetails {
 
   val id = "DeclarantDetails"
 
-  val mapping = Forms.mapping("details" -> EntityDetails.mapping)(DeclarantDetails.apply)(DeclarantDetails.unapply)
+  val declarantMapping = Forms
+    .mapping(
+      "eori" -> optional(
+        text()
+          .verifying("supplementary.eori.empty", nonEmpty)
+          .verifying("supplementary.eori.error", noLongerThan(17) and isAlphanumeric)
+      ),
+      "address" -> optional(Address.mapping)
```

Comment:
`"address" -> optional(Address.mapping)`
I would delete this line, as it is no longer necessary

Reply:
Good suggestion.  Done!